Edit: getting coins (#94)
FalixNodes-Software/help-center: _posts/falix/general/2021-11-22-getting-coins.md

1. Go to the [Join For Rewards](https://client.falixnodes.net/join_for_rewards) page.
2. You should see a list of servers on the right. Click on the "join" button next to a server to join it.

> 3. To receive coins for joining servers, reload the page. If you didn't get the reward, make sure to try relogging (logging out and back in).

Reviewer: ```suggestion
3. To receive coins for joining servers, reload the page. If you do not receive anything after reloading the page, make sure to try relogging (logging out and back in).
```

> 3. To receive coins for joining servers, reload the page. If you do not receive anything after reloading the page, make sure to try relogging (logging out and back in).

Reviewer: ```suggestion
3. To receive coins for joining servers, reload the page. If you still do not receive anything after reloading, make sure to try relogging (logging out and back in).
```
